flw/src/main/java/org/motechproject/nms/flw/domain/CallDetailRecord.java

```diff
@@ -0,0 +1,184 @@
+package org.motechproject.nms.flw.domain;
+
+import org.joda.time.DateTime;
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+
+import javax.jdo.annotations.Column;
+import javax.jdo.annotations.Extension;
+import javax.jdo.annotations.Order;
+import javax.jdo.annotations.Persistent;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
```

Comment:
Crap! That one always gets me.

Odd that it made it in since I thought I built that package so I could run
the tests in the api module.

On Tue, Apr 21, 2015 at 10:58 AM, Frank Huster <notifications@github.com>
wrote:

> In
> flw/src/main/java/org/motechproject/nms/flw/domain/CallDetailRecord.java
> <https://github.com/koshalt/mim/pull/65#discussion_r28804654>:
>
> > @@ -0,0 +1,184 @@
> > +package org.motechproject.nms.flw.domain;
> > +
> > +import org.joda.time.DateTime;
> > +import org.motechproject.mds.annotations.Entity;
> > +import org.motechproject.mds.annotations.Field;
> > +
> > +import javax.jdo.annotations.Column;
> > +import javax.jdo.annotations.Extension;
> > +import javax.jdo.annotations.Order;
> > +import javax.jdo.annotations.Persistent;
> > +import java.util.Collections;
> > +import java.util.List;
> > +import java.util.Set;
>
> unused import - fails the build.
>
> tsk tsk
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/koshalt/mim/pull/65/files#r28804654>.
>